src/Idea.js

```diff
@@ -64,15 +70,29 @@ export const Idea = ({
     onLinkEnd(id);
   }, [onLinkEnd, id]);
 
-  // if (isDragging) {
-  //   return <div ref={drag} />;
-  // }
+  const domElementRef = useCallback(
+    (domElementReference) => {
+      setDomElement(domElementReference);
+      drag(domElementReference);
+    },
+    [drag]
+  );
+
+  let selectedStyle = selected
+    ? {
+        border: "2px solid" + ideaPackage.selectedIdeaColor,
+        boxShadow: "4px 4px 15px" + ideaPackage.selectedIdeaColor,
```

Comment:
This concatenation will result in the string "2px solid#FFD700". This works only because CSS realizes what you're trying to do and fixes the mistake. These kinds of bugs are why I never use string concatenation. Instead I always prefer template strings.
```suggestion
        border: `2px solid ${ideaPackage.selectedIdeaColor}`,
        boxShadow: `4px 4px 15px ${ideaPackage.selectedIdeaColor}`,
```